react-components/src/components/RevealToolbar/MeasurementButton.tsx

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { useMemo, type ReactElement, useState } from 'react';
+import { useReveal } from '../RevealContainer/RevealContext';
+import { Button } from '@cognite/cogs.js';
+import { MeasurementTool } from '@cognite/reveal/tools';
+
+const distancesInFeetAndMeters = (distanceInMeters: number): string => {
+  const distanceInFeet = distanceInMeters * 3.281;
+  const distanceInFeetInt = Math.floor(distanceInFeet);
+  const distanceInches = Math.round(12 * (distanceInFeet - distanceInFeetInt));
```

Comment:
Pull out the magic numbers here to constants. I don't know what 3.281 or 12 mean here.